docs/learning-frontity/namespaces.md

```diff
@@ -207,7 +207,7 @@ export default {
 ```
 {% endcode %}
 
-But what if \(and now it is when this become interesting\) users don't want to use WordPress native comments but [Disqus](https://disqus.com/) comments?
+But what if \(and now it is when this becomes interesting\) users don't want to use WordPress native comments but [Disqus](https://disqus.com/) comments?
```

Comment:
'and now it is when this becomes interesting' would be better as 'and now this is where it becomes interesting' or even 'and now this is where things becomes interesting'